app/pages/index.js

```diff
@@ -0,0 +1,182 @@
+import Head from 'next/head'
```

Comment:
We needed this file because of Kubernetes health checks looking at `'/'`. In the future we may also decide to put visualizations, data exploration tools, etc.